blog.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Adoptadog</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+  <div class="header-container">
+  <header>
+    <div class="title">
+      <a href="index.html"><img src="images/adoptadog-logo.svg" alt="Adoptadog logo" class="logo"></a>
+      <h1>Seattle's premiere dog adoption agency</h1>
+    </div>
+    <div class="cart">
+      <div class="cartline1">
+        <img src="images/cart-icon.svg" alt="your checkout cart">
+        <span>$1234.56</span>
+      </div>
+      <h5 class="cartline2"><a href="checkout.html">Checkout</a></h5>
+    </div>
+  </header>
+  <div class="header-gradient"></div>
+    </div>
+
+  <nav class="topnav">
+    <h2>
+      <a href="index.html">Home</a> <span>/</span> <a href="adoption.html">Dogs</a> <span>/</span> <a href="blog.html">Blog</a>
+    </h2>
+  </nav>
+
+  <h3 class="headline">Adoptadog Blog</h3>
+  <section class="intro" id="blog-page">
+    <article class="post">
```

Comment:
I noticed your blog articles use the `overflow` css property. While this can be useful in some cases, it is generally not a good practice to use them unless absolutely necessary. In this case, it seems unnecessary to make your blog articles scroll horizontally when you should be making the content (image, text, etc.) fit the screen properly by making your <article> tags and their content use responsive sizes.

Reply:
Thanks again Tyler! I added a flexbox to the .intro CSS rule. I did notice however that the blog images on the blog page travel down the page as the browser size narrows (most noticeable at a browser width around 871px). As a result the images are mis-aligned with the corresponding type for each blog post. I'm not sure what is causing this thoughts?

Reply:
Thread resolved via email